tests/frontier/blockchain_st_test_helpers.py

```diff
@@ -207,6 +207,24 @@ def json_to_state(raw: Any) -> State:
     return state
 
 
+def load_test_transaction(
+    test_dir: str, test_file: str, network: str
```

Comment:
Since this is in the `tests/frontier` directory, this function will only ever be called for the `Frontier` network.

Would it make sense to make a file directly under `tests/`, deduplicate these functions there, and import it in each fork?

Reply:
Yes. That would make sense. I imagine there will be other such functions going forward, I will create a helpers package under `tests/` to house such functions.

Reply:
@SamWilsn On second thought, the function does use the `Transaction` class which is specific to a particular hard-fork. We could still create generic helpers outside the hard-fork specific test packages, but we will have to implement that with dynamic module imports. What do you think?

Reply:
@SamWilsn Have updated the PR